src/main/kotlin/io/perfometer/http/client/KtorHttpClient.kt

```diff
@@ -0,0 +1,27 @@
+package io.perfometer.http.client
+
+import io.ktor.client.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.util.*
+import io.perfometer.http.HttpRequest
+import io.perfometer.http.HttpResponse
+import io.perfometer.http.HttpStatus
+import java.net.URL
+
+class KtorHttpClient : HttpClient {
+
+    private val httpClient = HttpClient()
+
+    override suspend fun executeHttp(request: HttpRequest): HttpResponse {
+        val ktorResponse = httpClient.request<io.ktor.client.statement.HttpResponse>(
```

Comment:
You can use import alias here:

`import io.ktor.client.statement.HttpResponse as KtorHttpResponse`

Reply:
done